app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,21 @@
+class ReviewsController < ApplicationController
+  before_action :require_user
+  before_action :sanitize_params
+  def create
+    @video = Video.find(params[:video_id])
+    @review = @video.reviews.build(params.require(:review).permit(:rating, :body))
```

Comment:
I would push the `params.require....etc` to a private method and then I would use `@review = @video.reviews.build(review_params.merge!(user: current_user))` so you can skip the line after.